types/dropzone/index.d.ts

```diff
@@ -46,6 +46,12 @@ declare namespace Dropzone {
         upload?: DropzoneFileUpload;
     }
 
+    export interface DropZoneMockFile {
+        name: string;
+        size: number;
+        [index: string]: any;
```

Comment:
At least in our code base, we make heavy use of attaching additional data to either `DropzoneFile` or in this case, `DropzoneMockFile`. There are also a few answers to common problems solved by this. It's useful, since this file goes through all the callbacks which users of the library can override, so I added it this option here.

Alternatively, I assume, there's always the option to have `type MyMockFile = DropzoneMockFile & MyCustomFile`?

Reply:
I am fine with it
Unless extra property causes bugs, no point disallowing it

Reply:
TBH, I have the same problem with `DropzoneFile`. We attach some additional methods to it, for example for [checking the resolution](https://stackoverflow.com/a/30850495/1928168).

Would you, in general, be open to a PR adding additional fields to `DropzoneFile` as well? I was a bit hesitant, since in JS you can just attach anything everywhere, while we generally want more strict checks in TS.

Especially since Dropzone is very lax when accepting a "`file`". For a lot of operations, not every attribute is necessary.

Reply:
Unless the additional attributes would cause bugs
I am fine adding open ended signatures
But then I won't accept any specific non-official properties